mackup/applications/mps-youtube.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = MPS Youtube
+
+[configuration_files]
+.config/mps-youtube/config
+.config/mps-youtube/playlist_v2
```

Comment:
> I've seen some other config files without it so I'm leaving it as it is to match the current config files.

Not sure what you refer to. All the cfg files end with a newline in this repo. @vitorgalvao's link explains why it's important.

> but I want to know what @lra have to say about it

Just thanks @vitorgalvao for catching this.